ledger/daml-on-sql/src/test/suite/scala/on/sql/CliSpec.scala

```diff
@@ -42,6 +42,12 @@ class CliSpec
       config shouldEqual None
     }
 
+    "reject implicit party allocation" in {
+      val config =
+        cli.parse(Array("--ledgerid", "test-ledger", "--implicit-party-allocation", "true"))
```

Comment:
The parameter `--implicit-party-allocation` is not available for the DAML on SQL CLI. This parse probably returns `None` because it cannot find a definition of that parameter.

Reply:
Oh true, it is not parsed by CommonCli but by Sandbox next. Removed checks.